cmd/up/activate.go

```diff
@@ -308,7 +309,12 @@ func (up *upContext) waitUntilDevelopmentContainerIsRunning(ctx context.Context)
 		return err
 	}
 
+	to := time.Now().Add(up.Dev.Timeout.Resources)
+	var insufficientResourcesErr error
 	for {
+		if time.Now().After(to) && insufficientResourcesErr != nil {
+			return insufficientResourcesErr
```

Comment:
I would put a link to the docs to configure the timeout and explain what happened and how the user can configure a longer timeout if needed